src/index.html

```diff
@@ -2,20 +2,63 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
-      name="viewport"
-      content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+    <meta name="viewport" content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
-      "
-    >
+      ">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__iMac"
+        src="./images/imac.jpeg"
+        alt="Imac">
+
+      <p class="card__discription">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__productCode">
+        Product code: 195434
+      </p>
+
+      <div class="reviewSection reviewSection--Card-1">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="reviewSection__reviewRate">Reviews: 5</p>
+
+      </div>
+
+      <div class="price price--card-1">
+        <p class="price__text">
+          Price:
+        </p>
+        <p class="price__number">
+          $2,199
+        </p>
```

Comment:
`p` is not a correct tag here because semantically `p` means a complete piece of information but `Price:` is just a part of a message

So use `p` for entire block but here you need a `span`